src/lint/lint.nim

```diff
@@ -1,5 +1,55 @@
-proc lint* =
-  echo "No linting rules have yet been implemented.\n" &
-       "As we're in the process of implementing the linting rules, please " &
-       "re-run this command regularly to see if your track passes the " &
-       "latest linting rules."
+import std/[json, os, strformat, terminal]
+import ".."/[cli, helpers]
+
+template writeError(description: string, details: string) =
```

Comment:
At some point we probably want to move this as I expect that we'll be writing errors in other places too, but it is fine for now.